src/Analyzers/CSharp/Tests/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -1941,145 +2097,153 @@ class Program
+                    static void GetRef(int[] mem, int addr, int mode)
+                    {
+                        ref int i = ref addr;
+                        [|switch|] (mode)
+                        {
+                            case 0: i = true ? ref mem[mem[addr]] : ref mem[mem[addr]]; break;
+                            default: throw new Exception();
+                        }
+                    }
+                }
+                """,
+                """
+                using System;
+                class Program
+                {
+                    static void GetRef(int[] mem, int addr, int mode)
+                    {
+                        ref int i = ref addr;
+                        i = mode switch
+                        {
+                            0 => true ? ref mem[mem[addr]] : ref mem[mem[addr]],
+                            _ => throw new Exception(),
+                        };
+                    }
+                }
+                """);
         }
 
         [Fact]
         public async Task TopLevelStatement()
         {
-            var source = @"
-int i = 0;
-[|switch|] (i)
-{
-    case 1:
-        return 4;
-    default:
-        return 7;
-}";
-
-            var fixedSource = @"
-int i = 0;
-return i switch
-{
-    1 => 4,
-    _ => 7,
-};
-";
+            var source = """
+                int i = 0;
+                [|switch|] (i)
+                {
+                    case 1:
+                        return 4;
+                    default:
+                        return 7;
+                }
+                """;
+
+            var fixedSource = """
+                int i = 0;
+                return i switch
+                {
+                    1 => 4,
+                    _ => 7,
+                };
+
+                """;
 
             var test = new VerifyCS.Test
             {
-                TestCode = source,
+                TestState =
+                {
+                    OutputKind = OutputKind.ConsoleApplication,
+                    Sources = { source },
+                },
                 FixedCode = fixedSource,
                 LanguageVersion = LanguageVersion.CSharp9,
             };
 
-            test.ExpectedDiagnostics.Add(
-                // /0/Test0.cs(2,1): error CS8805: Program using top-level statements must be an executable.
-                DiagnosticResult.CompilerError("CS8805").WithSpan(2, 1, 2, 11));
-
             await test.RunAsync();
         }
 
         [Fact, WorkItem(44449, "https://github.com/dotnet/roslyn/issues/44449")]
         public async Task TopLevelStatement_FollowedWithThrow()
         {
             // We should be rewriting the declaration for 'j' to get 'var j = i switch ...'
-            var source = @"
-int i = 0;
-int j;
-[|switch|] (i)
-{
-    case 1:
-        j = 4;
-        break;
-    case 2:
-        j = 5;
-        break;
-}
-throw null;
-";
+            var source = """
+                int i = 0;
```

Comment:
❓ Why is a mechanical factoring PR containing code changes? this should have preserved the spacing.

Reply:
it's the mechanical approach that removes starting/ending lines, and remove indentation.  THat way we don't have tests that look like this:

```c#
var test = """

class C 
                        {
                               void Foo()
                               {
                               }
                        }
                   }

"""
```

But instead get the much more clearly desired:

```c#
var test = """
           class C 
           {
               void Foo()
               {
               }
           }
           """;
```